GECKODRIVER.md

```diff
@@ -0,0 +1,50 @@
+# How to use browsertime for evaluating Firefox TV
```

Comment:
I'm not sure we should include this file right now.   It looks like it needs some updating.

Reply:
Roger that.  There are several commits here that aren't ready; I won't land them.